### ISSUE TITLE
Don't know why python3 fails to detect the files of module

### ISSUE THREAD

**Issue by SacCrypt** [douban/linguist#30]
![image](https://user-images.githubusercontent.com/59285305/138109425-919664de-cc27-4ba6-ba4b-afdd6bb4e62f.png)

Don't know if this issue is relative or not

**Comment by jayvdb** [comment]
Yup, this package still uses Python 2-only style imports, so isnt compatible with Python 3 yet.
